drivers/performance_measurement/app/models/performance_measurement/report.rb

```diff
@@ -303,7 +303,8 @@ def can_see_client_details?(user)
               parts[:questions].each do |question|
                 report_client["#{variant_name}_#{question[:name]}"] = question[:value_calculation].call(member)
                 hud_project_ids.each do |project_id|
-                  project_clients << {
+                  project_clients[hud_client.id] ||= Set.new
+                  project_clients[hud_client.id] << {
```

Comment:
Maybe move this into a method so it doesn't have to get repeated every time? Not worth a class, but `add_client(project_clients, { ... })` maybe?

Reply:
Yeah, I had similar thoughts...  will do.